<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dia_imp_raw</class>
 <widget class="QDialog" name="dia_imp_raw">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>221</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>192</width>
    <height>221</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>192</width>
    <height>221</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttons">
   <property name="geometry">
    <rect>
     <x>-160</x>
     <y>180</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="box_x">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>38</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="box_y">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>68</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="box_z">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>98</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="box_le">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Little-Endian</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="box_sa">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Swap Axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Depth</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_dt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>8-Bit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16-Bit signed</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16-Bit unsigned</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32-Bit signed</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32-Bit unsigned</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32-Bit Float</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>64-Bit Float</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>11</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Type</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>dia_imp_raw</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>dia_imp_raw</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
